parse.c

```diff
@@ -1363,7 +1363,7 @@ int get_next_plain_string(void) {
       g_ss++;
       g_source_pointer++;
     }
-    else if ((c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || (c >= '0' && c <= '9') || c == '_' || c == '.' || c == '{' || c == '}' || c == '\\' || c == '@' || c == ':') {
+    else if ((unsigned char)c > 127 || (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || (c >= '0' && c <= '9') || c == '_' || c == '.' || c == '{' || c == '}' || c == '\\' || c == '@' || c == ':') {
```

Comment:
Please add a comment there telling people what's going on. And perhaps make c by default to be unsigned char? :) Otherwise loving how simple this fix is!

Reply:
On second thought I think I can polish this myself... :)